docs/source/tutorial/HOSVD.ipynb

```diff
@@ -0,0 +1,490 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Computing Tucker via the HOSVD\n",
+    "\n",
+    "```\n",
+    "Copyright 2022 National Technology & Engineering Solutions of Sandia,\n",
+    "LLC (NTESS). Under the terms of Contract DE-NA0003525 with NTESS, the\n",
+    "U.S. Government retains certain rights in this software.\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Higher-order Singular Value Decomposition (HOSVD) and Sequentially-truncased HOSVD (ST-HOSVD)\n",
+    "The HOSVD computes a Tucker decomposition of a tensor via a simple process. For each mode $k$, it computes the $r_k$ leading left singular values of the matrix unfolding and stores those as factor matrix $U_k$. Then it computes a `ttm` of the original tensor and all the factor matrices to yield the core of size $r_1 \\times r_2 \\times ... \\times r_d$. The core and factor matrices are used to form the `ttensor`. The values of $r_k$ that lead to a good approximation can be computed automatically to yield a specified error tolerance; this is recommended and the default in our code. The ST-HOSVD is an improvement on the HOSVD that does a TTM in *each* mode before moving on to the next mode. This has the advantage of shrinking the tensor at each step and reducing subsequent computations. ST-HOSVD is the default in the `hosvd` code.\n",
+    "\n",
+    "* L. R. Tucker, Some mathematical notes on three-mode factor analysis, Psychometrika, 31:279-311, 1966, http://dx.doi.org/10.1007/BF02289464\n",
+    "* L. D. Lathauwer, B. D. Moor and J. Vandewalle, A multilinear singular value decomposition, SIAM J. Matrix Analysis and Applications, 21(4):1253-1278, 2000, http://dx.doi.org/10.1137/S0895479896305696\n",
+    "* N. Vannieuwenhoven, R. Vandebril and K. Meerbergen, A New Truncation Strategy for the Higher-Order Singular Value Decomposition, SIAM J. Scientific Computing, 34(2):A1027-A1052, 2012, http://dx.doi.org/10.1137/110836067"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyttb as ttb\n",
+    "import numpy as np\n",
+    "\n",
+    "eps_machine = np.finfo(float).eps  # gets machine epsilon for floats, will be used later"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Simple example of usage"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create random 50 x 40 x 30 tensor with 5 x 4 x 3 core\n",
```

Comment:
# Create random tensor of shape (50, 40, 30) with core of shape (5, 4, 3)\n"

NOTE: change all other occurrences in the text of "size" to "shape" and use tuples to show the specific shapes.